Resources/Prototypes/_NF/Shipyard/Bison.yml.txt

```diff
@@ -0,0 +1,36 @@
+- type: vessel
+  id: Bison
+  name: NT Bison
+  description: A heavy-duty ship breaker with thickened armour plating and a substantial amount of living space. This is a top of the line commercial salvaging vessel that has minimal upkeep and can repressurise itself in the event of a hull breach. Recent events involving the ship's prior crew have led to it's liquidation and resale on the civilian market.
```

Comment:
these descriptions show up in tool-tips in game when you hover over the item in the shipyard console. Try to keep these much smaller so that they are more easily read.